src/sys/mod.rs

```diff
@@ -65,7 +65,7 @@ pub mod socket;
 
 pub mod stat;
 
-#[cfg(not(any(target_os = "freebsd", target_os = "netbsd")))]
+#[cfg(not(target_os = "netbsd"))]
```

Comment:
What's wrong with netbsd?

Reply:
Probably better to use a positive list here, not a negative list.  That's Nix's usual style, and this way you'll avoid breaking the build on platforms that don't have CI, like Dragonfly and emscripten.